doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -170,9 +170,15 @@ compatibility constraints.
 Adding new relations and morphisms
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. cmd::  Add Parametric Relation {* @binder } : (A t1 ... tn) (Aeq t′1 ... t′m) {? reflexivity proved by @term} {? symmetry proved by @term} {? transitivity proved by @term} as @ident
+.. cmd::  Add Parametric Relation {* @binder } : @one_term @one_term {? reflexivity proved by @one_term } {? symmetry proved by @one_term } {? transitivity proved by @one_term } as @ident
 
-   This command declares a parametric relation :g:`Aeq: forall (y1 : β1 ... ym : βm)`,
+   .. todo PR needs more work.  The first @one_term maps to :g:`relation (A t1 ... tn)`.
+      I guess the second maps to :g:`Aeq: forall (y1 : β1 ... ym : βm)`, but that
+      doesn't match the (Aeq t′1 ... t′m) in the cmd syntax before my changes.
+      And then not sure how :g:`(A : αi -> ... αn -> Type)` fits in at all, e.g.
+      what is α?
```

Comment:
@mattam82 Can you help (or designate someone that could help)?

Reply:
The first term should be a Type (the carrier of the relation), say T, while the second should be a relation on T, i.e. of type (T -> T -> Prop). As written before, we had A and Aeq applied to some terms built from the binders. So `A: forall x1 ... x2, Type` and `Aeq : forall x1 .. x2, (A x1 ... xn) -> A (x1 ... xn) -> Prop)`, or equivalently, `Aeq : forall x1 ... xn, relation (A x1 ... xn)`.

Reply:
Here's my best shot:

![image](https://user-images.githubusercontent.com/1253341/96677706-01cf1b80-1325-11eb-8696-fd50be585a92.png)

How do the binders relate to the alphas?  Or are they unrelated?

Reply:
The binders are usually the same as the alphas, but there is no strict relationship. Maybe rather say that `one_termA` and `one_term_Aeq` should be typeable under the context `binders` instead.